tests/test_sample_rayservice_yamls.py

```diff
@@ -76,7 +76,7 @@ def __init__(
         self,
         custom_resource_object,
         rulesets: List[RuleSet] = [],
-        timeout: int = 90,
+        timeout: int = 180,
```

Comment:
We now need to wait for the old RayCluster to be deleted if switching the RayCluster. The old RayCluster will only be deleted 60 seconds after it becomes inactive. Therefore, we need at least 90 + 60 = 150 seconds; the extra 30 seconds are for the overhead and safety.